kube/services/jobs/usersync-cronjob.yaml

```diff
@@ -74,6 +82,21 @@ spec:
                   configMapKeyRef:
                     name: manifest-global
                     key: sync_from_dbgap
+                    optional: true
+              - name: DBGAP_BACKUP_BUCKET
+                valueFrom:
+                  configMapKeyRef:
+                    name: manifest-global
+                    key: dbgap_backup_bucket
+                    optional: true
+              - name: AWS_ROLE_ARN
+                valueFrom:
```

Comment:
AWS_ROLE_ARN and AWS_WEB_IDENTITY_TOKEN_FILE are also auto-magically setup by EKS - don't need these 2